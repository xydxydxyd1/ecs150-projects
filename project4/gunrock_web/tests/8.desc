Run ds3bits on our two given disk images
