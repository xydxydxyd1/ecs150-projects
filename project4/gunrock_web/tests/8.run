./ds3bits tests/disk_images/big_directory.img
