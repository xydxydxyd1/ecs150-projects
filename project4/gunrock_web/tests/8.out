Super
inode_region_addr 3
inode_region_len 2
num_inodes 64
data_region_addr 5
data_region_len 128
num_data 128

Inode bitmap
255 255 255 255 255 255 15 0 

Data bitmap
255 255 255 255 255 255 31 0 0 0 0 0 0 0 0 0 
